dodo.py

```diff
@@ -765,3 +765,10 @@ def task_tidy():
             'find . | grep -E "(__pycache__|\.pyc$)" | xargs rm -rf',
         ],
     }
+
+def task_dynamo():
```

Comment:
nothing depends on this task?

Reply:
@scottidler added depends, please review.